### PR TITLE
create sprint-26-summary.md

### PR DESCRIPTION
**Pull request by amilash**
create sprint-26-summary.md and populate based on status as of 08032021 10:21am PST

### Summary of Changes
documentation only

### How to Test
N/A

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [ ] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [ ] Check against the criteria:

As facilitator/product manager, will decide if ACs are met from Raft's perspective.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [ ] Frontend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
  + [ ] Backend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

+ [ ] Are backend code style checks passing on CircleCI?
+ [ ] Are frontend code style checks passing on CircleCI?
+ [ ] Does this PR change any linting or CI settings?

### Deliverable 4: Accessible

> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*

> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*

+ [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful build with a single command*

> NOTE: until we have a proper staging environment this may not be satisfiable prior to merging

+ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

+ [ ] If this PR introduces backend code, is that code documented both inline and overall?
+ [ ] If this PR introduces frontend code, is that code documented both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

+ [ ] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any security issues?

### CODE REVIEW

File: docs/Sprint-Review/sprint-26-summary.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- As an OFA Admin, I want an accessible, 508-compliant user interface for managing permissions [#892](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/892)
+- [Backend] Utilize Django settings modules for non-sensitive environment variables that differ between deployed environments [#970](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/970)
+- Round 7 (Sprint 24-26) UX Research and Design#1017
+
+**Current Sprint Backlog**
+- As a dev, I need to know which authentication service we're using (login.gov vs. NextGen XMS) [#638](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/638)
+- As a TDP user, I want to see a picture that better represents the program [#640](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/640)
+
+
+## Agenda for Sprint 25 Demo
```
Comment (ADPennington) [inline docs/Sprint-Review/sprint-26-summary.md:71]:
```suggestion
## Agenda for Sprint 26 Demo 
```

File: docs/Sprint-Review/sprint-26-summary.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Blockers + Authentication Request Workflow
+
+
+## Sprint 26 Goals
+
+- Finish closing ATO functionality tickets
+- Focus on Secret Key Leakage Mitigation Epic
+- UX Ideation for Parsing Blocker Communications
+
+**Next Sprint (27) Goals:**
```
Comment (lfrohlich) [inline docs/Sprint-Review/sprint-26-summary.md:18]:
```suggestion
[**Next Sprint (27) Goals:**](https://github.com/raft-tech/TANF-app/milestone/30) 
```

File: docs/Sprint-Review/sprint-26-summary.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+## Agenda for Sprint 26 Demo 
+- As a dev, I want an automated tool to prevent me from committing secret keys to the repo [#965](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/965) Andrew
+- [DevOps] Generate a new, random DJANGO_SECRET_KEY on initial Cloud.gov deployments or rebuilds [#967](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/967) - Andrew/John
+- (From Last Sprint) Perform scheduled OWASP scans against deployed site(s)[#1032](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1032)- Aaron
+- (UX Sprint 26) Test Error Communication for Parsing Blockers [#1020](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1020) - Miles/Dmitri
+- Round 7 (Sprint 24-26) UX Research and Design [#1017](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1017)
+- Django Admin a11y Fixes (Sprint 3) [#1054](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1054)
```
Comment (lfrohlich) [inline docs/Sprint-Review/sprint-26-summary.md:77]:
```suggestion

```

File: docs/Sprint-Review/sprint-26-summary.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+- Round 7 (Sprint 24-26) UX Research and Design#1017
+
+**Current Sprint Backlog**
+- As a dev, I need to know which authentication service we're using (login.gov vs. NextGen XMS) [#638](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/638)
+- As a TDP user, I want to see a picture that better represents the program [#640](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/640)
+
+
+## Agenda for Sprint 26 Demo 
+- As a dev, I want an automated tool to prevent me from committing secret keys to the repo [#965](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/965) Andrew
+- [DevOps] Generate a new, random DJANGO_SECRET_KEY on initial Cloud.gov deployments or rebuilds [#967](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/967) - Andrew/John
```
Comment (lfrohlich) [inline docs/Sprint-Review/sprint-26-summary.md:73]:
```suggestion
```

**Comment by amilash** [comment]
ready for review @lfrohlich @ADPennington

**Review by amilash — COMMENTED** [review]
Ready for approval. all comments resolved.

**Review by lfrohlich — APPROVED** [review]
👍

**Pull request merged**